Oct 13, 2013 12:09:05 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: contextInitialized()
Oct 13, 2013 12:09:05 AM org.apache.catalina.core.ApplicationContext log
INFO: SessionListener: contextInitialized()
Oct 13, 2013 12:09:05 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: attributeAdded('org.apache.jasper.compiler.TldLocationsCache', 'org.apache.jasper.compiler.TldLocationsCache@25dafe28')
Oct 13, 2013 12:12:03 AM org.apache.catalina.core.ApplicationContext log
INFO: SessionListener: contextDestroyed()
Oct 13, 2013 12:12:03 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: contextDestroyed()
Oct 13, 2013 12:12:19 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: contextInitialized()
Oct 13, 2013 12:12:19 AM org.apache.catalina.core.ApplicationContext log
INFO: SessionListener: contextInitialized()
Oct 13, 2013 12:12:19 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: attributeAdded('org.apache.jasper.compiler.TldLocationsCache', 'org.apache.jasper.compiler.TldLocationsCache@740227c')
Oct 13, 2013 12:17:10 AM org.apache.catalina.core.ApplicationContext log
INFO: Warning: no max runtimes specified.
Oct 13, 2013 12:17:11 AM org.apache.catalina.core.ApplicationContext log
INFO: jruby 1.6.6 (ruby-1.8.7-p357) (2012-01-30 5673572) (Java HotSpot(TM) 64-Bit Server VM 1.7.0_40) [darwin-x86_64-java]
Oct 13, 2013 12:17:11 AM org.apache.catalina.core.ApplicationContext log
INFO: Info: using runtime pool timeout of 30 seconds
Oct 13, 2013 12:17:11 AM org.apache.catalina.core.ApplicationContext log
INFO: Warning: no min runtimes specified.
Oct 13, 2013 12:17:11 AM org.apache.catalina.core.ApplicationContext log
INFO: Warning: no max runtimes specified.
Oct 13, 2013 12:17:29 AM org.apache.catalina.core.ApplicationContext log
INFO: Connecting to database specified by database.yml

Oct 13, 2013 12:17:29 AM org.apache.catalina.core.ApplicationContext log
INFO: Warning: no min runtimes specified.
Oct 13, 2013 12:17:29 AM org.apache.catalina.core.ApplicationContext log
INFO: Warning: no max runtimes specified.
Oct 13, 2013 12:17:29 AM org.apache.catalina.core.ApplicationContext log
INFO: An exception happened during JRuby-Rack startup
/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/initializers/session_store.rb:3: syntax error, unexpected ':'

MedditJrubyTest::Application.config.session_store :cookie_store, key: '_medditJrubyTest_session'
                                                                    ^
--- System
jruby 1.6.6 (ruby-1.8.7-p357) (2012-01-30 5673572) (Java HotSpot(TM) 64-Bit Server VM 1.7.0_40) [darwin-x86_64-java]
Time: Sun Oct 13 00:17:29 -0700 2013
Server: Apache Tomcat/7.0.42
jruby.home: file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-stdlib-1.6.6.jar!/META-INF/jruby.home

--- Context Init Parameters:
public.root = /
rails.env = production

--- Backtrace
SyntaxError: /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/initializers/session_store.rb:3: syntax error, unexpected ':'

MedditJrubyTest::Application.config.session_store :cookie_store, key: '_medditJrubyTest_session'
                                                                    ^
              load at org/jruby/RubyKernel.java:1052
              load at /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/initializers/session_store.rb:245
   load_dependency at /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/gems/gems/activesupport-3.2.14/lib/active_support/dependencies.rb:236
              load at /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/gems/gems/activesupport-3.2.14/lib/active_support/dependencies.rb:245
            Engine at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/engine.rb:593
              each at org/jruby/RubyArray.java:1614
            Engine at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/engine.rb:592
     instance_exec at org/jruby/RubyKernel.java:2060
               run at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/initializable.rb:30
  run_initializers at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/initializable.rb:55
              each at org/jruby/RubyArray.java:1614
  run_initializers at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/initializable.rb:54
       initialize! at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/application.rb:136
              send at org/jruby/RubyKernel.java:2074
    method_missing at /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/railtie/configurable.rb:30
            (root) at /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/environment.rb:5
           require at org/jruby/RubyKernel.java:1027
  load_environment at /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/environment.rb:167
            to_app at file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-rack-1.1.3.jar!/jruby/rack/rails.rb:172
               new at file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-rack-1.1.3.jar!/jruby/rack/rails.rb:192
            (root) at <web.xml>:1
     instance_eval at org/jruby/RubyKernel.java:2043
        initialize at file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-rack-1.1.3.jar!/vendor/rack-1.3.6/rack/builder.rb:51
            (root) at <web.xml>:1

--- RubyGems
Gem.dir: /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9
Gem.path:
/Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9
/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/gems
Activated gems:
  bundler-1.3.5
  rake-10.1.0
  i18n-0.6.5
  multi_json-1.8.1
  activesupport-3.2.14
  builder-3.0.4
  activemodel-3.2.14
  erubis-2.7.0
  journey-1.0.4
  rack-1.4.5
  rack-cache-1.2
  rack-test-0.6.2
  hike-1.2.3
  tilt-1.4.1
  sprockets-2.2.2
  actionpack-3.2.14
  mime-types-1.25
  polyglot-0.3.3
  treetop-1.4.15
  mail-2.5.4
  actionmailer-3.2.14
  arel-3.0.2
  tzinfo-0.3.38
  activerecord-3.2.14
  activerecord-jdbc-adapter-1.3.2
  jdbc-sqlite3-3.7.2.1
  activerecord-jdbcsqlite3-adapter-1.3.2
  activeresource-3.2.14
  bouncy-castle-java-1.5.0147
  coffee-script-source-1.6.3
  execjs-2.0.2
  coffee-script-2.2.0
  rack-ssl-1.3.3
  json-1.8.0-java
  rdoc-3.12.2
  thor-0.18.1
  railties-3.2.14
  coffee-rails-3.2.2
  jquery-rails-3.0.4
  jruby-openssl-0.9.2
  rails-3.2.14
  sass-3.2.12
  sass-rails-3.2.6
  therubyrhino_jar-1.7.4
  therubyrhino-2.0.2
  uglifier-2.2.1

--- Bundler
Bundler.bundle_path: /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9
Bundler.root: /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF
Gemfile: Gemfile
Settings:
  without = development:test
  gemfile = Gemfile
  bin_path = /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/bundler-1.3.5/bin/bundle

--- JRuby-Rack Config
compat_version = 
filter_adds_html = true
filter_verifies_resource = false
ignore_environment = false
initial_runtimes = 
jms_connection_factory = 
jms_jndi_properties = 
logger = org.jruby.rack.logging.ServletContextLogger@70c3c5f8
logger_class_name = servlet_context
logger_name = jruby.rack
maximum_runtimes = 
num_initializer_threads = 
rackup = 
rackup_path = 
rewindable = true
runtime_arguments = 
runtime_timeout_seconds = 
serial_initialization = false
servlet_context = org.apache.catalina.core.ApplicationContextFacade@367a9db8

Oct 13, 2013 12:17:29 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
org.jruby.rack.RackInitializationException: /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/initializers/session_store.rb:3: syntax error, unexpected ':'

MedditJrubyTest::Application.config.session_store :cookie_store, key: '_medditJrubyTest_session'
                                                                    ^
	from /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/initializers/session_store.rb:245:in `load'
	from /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/gems/gems/activesupport-3.2.14/lib/active_support/dependencies.rb:236:in `load_dependency'
	from /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/gems/gems/activesupport-3.2.14/lib/active_support/dependencies.rb:245:in `load'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/engine.rb:593:in `Engine'
	from org/jruby/RubyArray.java:1614:in `each'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/engine.rb:592:in `Engine'
	from org/jruby/RubyKernel.java:2060:in `instance_exec'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/initializable.rb:30:in `run'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/initializable.rb:55:in `run_initializers'
	from org/jruby/RubyArray.java:1614:in `each'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/initializable.rb:54:in `run_initializers'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/application.rb:136:in `initialize!'
	from org/jruby/RubyKernel.java:2074:in `send'
	from /Applications/jrubystack-1.7.5-0/jruby/lib/ruby/gems/1.9/gems/railties-3.2.14/lib/rails/railtie/configurable.rb:30:in `method_missing'
	from /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/environment.rb:5:in `(root)'
	from org/jruby/RubyKernel.java:1027:in `require'
	from /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/environment.rb:167:in `load_environment'
	from file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-rack-1.1.3.jar!/jruby/rack/rails.rb:172:in `to_app'
	from file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-rack-1.1.3.jar!/jruby/rack/rails.rb:192:in `new'
	from <web.xml>:1:in `(root)'
	from org/jruby/RubyKernel.java:2043:in `instance_eval'
	from file:/Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/lib/jruby-rack-1.1.3.jar!/vendor/rack-1.3.6/rack/builder.rb:51:in `initialize'
	from <web.xml>:1:in `(root)'

	at org.jruby.rack.DefaultRackApplicationFactory$4.init(DefaultRackApplicationFactory.java:206)
	at org.jruby.rack.DefaultRackApplicationFactory.getApplication(DefaultRackApplicationFactory.java:56)
	at org.jruby.rack.PoolingRackApplicationFactory.getApplication(PoolingRackApplicationFactory.java:95)
	at org.jruby.rack.DefaultRackDispatcher.getApplication(DefaultRackDispatcher.java:20)
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:29)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:312)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
Caused by: org.jruby.exceptions.RaiseException: (SyntaxError) /Applications/jrubystack-1.7.5-0/apache-tomcat/webapps/medditJrubyTest/WEB-INF/config/initializers/session_store.rb:3: syntax error, unexpected ':'

MedditJrubyTest::Application.config.session_store :cookie_store, key: '_medditJrubyTest_session'
                                                                    ^

Oct 13, 2013 12:17:47 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:17:54 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:18:19 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:18:37 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:18:39 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:18:39 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:18:42 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:312)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:18:47 AM org.apache.catalina.core.ApplicationContext log
SEVERE: Application Error
java.lang.NullPointerException
	at org.jruby.rack.AbstractRackDispatcher.process(AbstractRackDispatcher.java:30)
	at org.jruby.rack.AbstractFilter.doFilter(AbstractFilter.java:31)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:243)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:210)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:222)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:123)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:502)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:171)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:99)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:953)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:118)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:408)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1023)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:589)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:310)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:31:41 AM org.apache.catalina.core.ApplicationFilterConfig release
SEVERE: Failed to destroy the filter named [RackFilter] of type [org.jruby.rack.RackFilter]
java.lang.NullPointerException
	at org.jruby.rack.PoolingRackApplicationFactory.destroy(PoolingRackApplicationFactory.java:120)
	at org.jruby.rack.DefaultRackDispatcher.destroy(DefaultRackDispatcher.java:43)
	at org.jruby.rack.AbstractFilter.destroy(AbstractFilter.java:38)
	at org.apache.catalina.core.ApplicationFilterConfig.release(ApplicationFilterConfig.java:315)
	at org.apache.catalina.core.StandardContext.filterStop(StandardContext.java:4811)
	at org.apache.catalina.core.StandardContext.stopInternal(StandardContext.java:5618)
	at org.apache.catalina.util.LifecycleBase.stop(LifecycleBase.java:232)
	at org.apache.catalina.core.ContainerBase.removeChild(ContainerBase.java:1028)
	at org.apache.catalina.startup.HostConfig.deleteRedeployResources(HostConfig.java:1299)
	at org.apache.catalina.startup.HostConfig.checkResources(HostConfig.java:1250)
	at org.apache.catalina.startup.HostConfig.check(HostConfig.java:1439)
	at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:315)
	at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:119)
	at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:90)
	at org.apache.catalina.core.ContainerBase.backgroundProcess(ContainerBase.java:1374)
	at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1530)
	at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1540)
	at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.run(ContainerBase.java:1519)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:31:41 AM org.apache.catalina.core.StandardContext listenerStop
SEVERE: Exception sending context destroyed event to listener instance of class org.jruby.rack.rails.RailsServletContextListener
java.lang.NullPointerException
	at org.jruby.rack.PoolingRackApplicationFactory.destroy(PoolingRackApplicationFactory.java:120)
	at org.jruby.rack.RackServletContextListener.contextDestroyed(RackServletContextListener.java:56)
	at org.apache.catalina.core.StandardContext.listenerStop(StandardContext.java:4980)
	at org.apache.catalina.core.StandardContext.stopInternal(StandardContext.java:5626)
	at org.apache.catalina.util.LifecycleBase.stop(LifecycleBase.java:232)
	at org.apache.catalina.core.ContainerBase.removeChild(ContainerBase.java:1028)
	at org.apache.catalina.startup.HostConfig.deleteRedeployResources(HostConfig.java:1299)
	at org.apache.catalina.startup.HostConfig.checkResources(HostConfig.java:1250)
	at org.apache.catalina.startup.HostConfig.check(HostConfig.java:1439)
	at org.apache.catalina.startup.HostConfig.lifecycleEvent(HostConfig.java:315)
	at org.apache.catalina.util.LifecycleSupport.fireLifecycleEvent(LifecycleSupport.java:119)
	at org.apache.catalina.util.LifecycleBase.fireLifecycleEvent(LifecycleBase.java:90)
	at org.apache.catalina.core.ContainerBase.backgroundProcess(ContainerBase.java:1374)
	at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1530)
	at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.processChildren(ContainerBase.java:1540)
	at org.apache.catalina.core.ContainerBase$ContainerBackgroundProcessor.run(ContainerBase.java:1519)
	at java.lang.Thread.run(Thread.java:724)

Oct 13, 2013 12:32:07 AM org.apache.catalina.core.ApplicationContext log
INFO: SessionListener: contextDestroyed()
Oct 13, 2013 12:32:07 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: contextDestroyed()
Oct 13, 2013 12:32:22 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: contextInitialized()
Oct 13, 2013 12:32:22 AM org.apache.catalina.core.ApplicationContext log
INFO: SessionListener: contextInitialized()
Oct 13, 2013 12:32:22 AM org.apache.catalina.core.ApplicationContext log
INFO: ContextListener: attributeAdded('org.apache.jasper.compiler.TldLocationsCache', 'org.apache.jasper.compiler.TldLocationsCache@64974bf0')
